app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,42 @@
+<footer class="footer">
+  <section>
+    <div class="footer__heading">Questions or feedback</div>
+    <p>Spotted a typo? Something missing in docs?</p>
+    <ul class="footer__list">
+      <li class="footer__list-item">
+        <a class="footer__list-item-link" href="https://github.com/buildkite/docs/issues">
+          <img alt="Icon: issue" class="footer__list-item-icon" src="<%= image_url('icons/issue.svg') %>" />
+          Open an issue
+        </a>
+      </li>
+      <li class="footer__list-item">
+        <a class="footer__list-item-link" href="<%= open_source_url %>">
+          <img alt="Icon: pull request" class="footer__list-item-icon" src="<%= image_url('icons/pull-request.svg') %>" />
+          Contribute an update
+        </a>
+      </li>
+    <ul>
+  </section>
+  <section>
+    <div class="footer__heading">Reach out</div>
+    <p>We'd love to know what you think!</p>
+    <ul class="footer__list">
+      <li class="footer__list-item">
+        <a class="footer__list-item-link" href="https://chat.buildkite.community/">
+          <img alt="Icon: Slack" class="footer__list-item-icon" src="<%= image_url('icons/slack.svg') %>" />
+          <% unless "test-analytics".in? request.path %>
+            Join the community Slack channel
```

Comment:
Very minor suggestion: "Join the Buildkite community Slack" for the default

Reply:
pushed a commit to fix :)